### PR TITLE
Modular

### PR DESCRIPTION
**Pull request by pirat89**
Note: I am thinking whether to sort the songs already into the specific subdirs yet, grouped according to the artist, genre, ... what do you think, @ingvagabund? And thinking even about the move of the pdf to the separate dir (or just rename it) and re-generate it only one upon the time. Instead of change during every commit/PR.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,9 @@
 generate:
-	pdflatex songbook.tex
+	@cat `find songs -name '*.tex'` > songs.tex
```
Comment (ingvagabund) [inline Makefile]: This will generate the list randomly. `find songs -name '*.tex' | sort -u` at least

File: songs/bedna_od_whisky.tex [hunk 2]
```diff
@@ -0,0 +1,64 @@
+\beginsong{Bedna od Whisky}[by={Wanastovi vjeci},
```
Comment (ingvagabund) [inline songs/bedna_od_whisky.tex]: I would stick with `songs/<interpret>-<song>.tex` pattern

File: Makefile [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+	@pdflatex songbook.tex
 
 clean:
-	rm -f *.aux *.log
+	rm -f *.aux *.log songs.tex
```
Comment (ingvagabund) [inline Makefile]: You can keep the `songs.tex` even if it is generated.

File: Makefile (hunk 1 above)
Comment (pirat89) [reply Makefile]: You are right

File: songs/bedna_od_whisky.tex (hunk 2 above)
Comment (pirat89) [reply songs/bedna_od_whisky.tex]: Instead of that, I am more thinking about `songs/<interpret>/<song>.tex`

File: Makefile (hunk 3 above)
Comment (pirat89) [reply Makefile]: I am thinking here about the `.gitignore`

File: songs/bedna_od_whisky.tex (hunk 2 above)
Comment (ingvagabund) [reply songs/bedna_od_whisky.tex]: SGTM

File: Makefile (hunk 1 above)
Comment (dhodovsk) [reply Makefile]: Could it be possible to order it by time somehow (maybe by file commit history?)? To deal with cases the playlist is extended after print?

Comment (ingvagabund) [reply Makefile]: That will not work if we build the pdf from a tarball. But we could check a create timestamp of each file.

Comment (dhodovsk) [reply Makefile]: I believe that would have different results for different users (local timestamp of file is set when it's been cloned or rebased). Or did you mean something else?

Comment (dhodovsk) [reply Makefile]: Maybe this could help: `git log --format=%aD <songfile> | tail -1`

File: songs/bedna_od_whisky.tex (hunk 2 above)
Comment (pirat89) [reply songs/bedna_od_whisky.tex]: done

File: Makefile (hunk 1 above)
Comment (ingvagabund) [reply Makefile]: It's unpleasant the pages gets reordered each time we add new songs. However, that's inevitable if we sort the songs by interpret. I would not bother with that much.

**Comment by pirat89** [comment]
I found that I used sort completely in different place where I should use it *shame* So I fixed that. Now it could be everything ok according to discussion above.

**Comment by dhodovsk** [comment]
LGTM - can be merged, @ingvagabund ?

**Pull request merged**